### ISSUE TITLE
This is a lie

### ISSUE THREAD

**Issue by Kuchiriel** [buefy/nuxt-buefy#7]
Usage
{
  modules: [
    // Simple usage
    'nuxt-buefy',

    ['nuxt-buefy', { /* buefy options */ }]
 ]
}

It isn't that simple, and this repo is the proof.

**Comment by anteriovieira** [comment]
Hi @Kuchiriel , I did not quite understand your doubt, could you rephrase the question?

**Comment by Kuchiriel** [comment]
It doesn't work out of the box, you have to @import '~/node_modules/buefy/lib/buefy.min.css' to make it work. There is no difference using nuxt-buefy or buefy at all.

**Comment by anteriovieira** [comment]
> Hi @anteriovieira , It doesn't work out of the box, you have to @import ...

Should be working,  the css is automatically added according to implementation https://github.com/buefy/nuxt-buefy/blob/master/lib/module.js#L21

I'll do some testing later, if you have any progress let me know.

**Comment by Kuchiriel** [comment]
No progress, I am justing using bulma now. 

nuxt.config.js
`  modules: [
    // provide path to the file with resources
    ['nuxt-sass-resources-loader', './assets/main.scss']
  ],`

main.scss
`@import "~bulma/sass/navbar.sass`

**Comment by damien-biasotto** [comment]
@Kuchiriel , Well I just  followed the README.md, added only `nuxt-buefy` added to `modules` section of the `nuxt.config.js` file and it seems to work out of the box.

**Comment by Kuchiriel** [comment]
You cloned the repo or started from scratch?

**Comment by damien-biasotto** [comment]
Sorry @Kuchiriel for the delay, I totally missed the notificiation.

I am not sure what you're implying by started from scratch but here what I've done :

* I setup a basic nuxt app
* added nuxt-buefy (using npm) 
* added nuxt-buefy to nuxt modules.

And that's it. 

My node version at that time was 10.4.0, 
npm version was (probably) 6.1.0,
nuxt version 1.0
nuxt-buefy: 0.0.3

I will try to set it up again, just to double check if I missed something, and if it's working I'll zip that and share it with you

**Comment by damien-biasotto** [comment]
@Kuchiriel so I started from scratch : 

The commands I used:

- `mkdir nuxt-buefy-debug;`
- `cd $_; `
- `vue init nuxt-community/starter-template .`
- `npm i`
- `npm i -S nuxt-buefy`

Environment:

- `npm`: 6.1.0
- `node`: 10.4.1


And here is my project  https://nofile.io/f/OKFbc6WBTUp/nuxt-buefy-debug.zip
: 
https://nofile.io/f/OKFbc6WBTUp/nuxt-buefy-debug.zip

**Comment by Kuchiriel** [comment]
`npm i -S nuxt-buefy` This is the same as cloning the project, but the files will be in node_modules folder.

**Comment by fraunos** [comment]
@Kuchiriel well, maybe it's not? Does simple cloning create buefy.js file in .nuxt directory?
